setup.py

```diff
@@ -2,6 +2,7 @@
 import os
 import subprocess
 import sys
+from distutils.sysconfig import customize_compiler
```

Comment:
It's maybe a concern for only in the future, but distutils is kind of deprecated (I know we also still use it for version comparison ..), and they are probably going to actually deprecate in the next python version. For `distutils.sysconfig` they refer to the stdlib `sysconfig` package as alternative, but that doesn't seem to have this function.

So if possible, I would maybe prefer to stay away from such customization. Do you know if the other SO answer about setting an ENV variable worked?

Reply:
The ENV approach did not work.

`setuptools.command.build_ext` directly uses `distutils.sysconfig.customize_compiler` internally.  

Since it looks like the [PEP deprecated distutils](https://www.python.org/dev/peps/pep-0632/) is not yet final, perhaps this is something we should revisit once it becomes final?

I added a note here just to give us a head's up.